src/app/public/modules/title/title.service.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  Injectable
+} from '@angular/core';
+
+import {
+  Title
+} from '@angular/platform-browser';
+
+import {
+  SkyAppSetTitleArgs
+} from './set-title-args';
+
+/**
+ * Provides a method for setting a formatted title on the current window.
+ */
+@Injectable()
+export class SkyAppTitleService {
+
+  constructor(private title: Title) { }
+
+  /**
+   * Sets the title on the current window.
+   * @param args An array of title parts. The parts will be concatenated with a hyphen between
+   * each part.
+   */
+  public setTitle(args: SkyAppSetTitleArgs): void {
```

Comment:
So when does this get called vs when does the auth client `setTitle` get called? Is it just for testing or would it be called any time the auth client service isn't provided? Is there any time the auth client service wouldn't be provided?

Reply:
If the SPA doesn't have the omnibar enabled, the alternate implementation that passes through to omnibar would not be injected, and this method would get called.

Reply:
Here's the code that will go into builder once this is released:

```
  if (skyAppConfig.skyux.omnibar) {
    nodeModuleImports.push(`import {
  SkyAppOmnibarTitleService
} from '${skyAppConfig.runtime.runtimeAlias}/omnibar/omnibar-title.service';`);

    runtimeProviders.push(`{ provide: SkyAppTitleService, useClass: SkyAppOmnibarTitleService }`);
  } else {
    runtimeProviders.push('SkyAppTitleService');
  }
```